src/account/account.ts

```diff
@@ -59,3 +81,30 @@ export async function createUser(
 
   return { ...account, reference }
 }
+
+/**
+ * Creates a new user (version 2) and uploads the encrypted account to the network
+ *
+ * @param connection connection information for data uploading
+ * @param username FDP username
+ * @param password FDP password
+ * @param mnemonic mnemonic phrase
+ */
+export async function createUser(
+  connection: Connection,
+  username: string,
+  password: string,
+  mnemonic: string,
+): Promise<Reference> {
+  const { wallet, encryptedMnemonic } = await createUserAccount(password, mnemonic)
+  const topic = generateRandomBase64String(RANDOM_TOPIC_LENGTH)
+  const encryptedMnemonicAddress = removeZeroFromHex(
+    await writeFeedData(connection, topic, stringToBytes(encryptedMnemonic), wallet.privateKey),
```

Comment:
I think we need to pad with random data after `encryptedMnemonic` for the whole chunk as you do with its reference